### ISSUE TITLE
Looking for TESLA 43T319BF TV remote preset

### ISSUE THREAD

**Issue by zorgoz** [irplus-remote/irplus-codes.github.io#525]
The remote I am trying to emulate is told to be compatible with following models (according to an ebay seller): TESLA 32T319BH, 32T319BHS, 32T319SH, 40T319SF, 40T319BF, 43T319SF, 43T319BF, STRONG SRT24HX4003, SRT32HX4003, SRT43FX4003, SRT49FX4003, SRT55FX4003, SRT65FX4003, UNITED 42SK30, HOF15K034GPD21, LED32SK30, LED43SK30, 49S306BF, INNO HIT IH22E2000, TESLA 32S306BH
I have tried Changhong and Strong presets in the app (and other similar apps too) but none worked.

**Comment by irplus-remote** [comment]
Yea, exactly this was where I got the information from. Can you confirm the genuine remove you have is working with that TV set? And that irplus is working with your smartphone with a different device just to exclude any other issue. (Please tell me the Brand and Smartphone Model as well)

**Comment by zorgoz** [comment]
Yes, absolutely. The genuine remote is working and it looks exactly like the photo on ebay. And I have a set-top-box from UPC, that I can perfectly control with the app from my POCO f2 pro (also from the MI remote control app).

**Comment by irplus-remote** [comment]
There seem to exist two codesets for some of those tv. Can you try this code (this is power only just to test)

```xml
<irplus>
    <device manufacturer="TESLA" model="TEST" columns="12" format="PRONTO_HEX">
        <button label="PWR" span="4">0000 006D 0000 0022 00AB 00AB 0015 0015 0015 0040 0015 0040 0015 0040 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0040 0015 0040 0015 0040 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0040 0015 0040 0015 0015 0015 0015 0015 0015 0015 0015 0015 0040 0015 0040 0015 0015 0015 0015 0015 0040 0015 0040 0015 0040 0015 0040 0015 06F6</button>

    </device>
</irplus>

```

**Comment by zorgoz** [comment]
No luck 😔

**Comment by irplus-remote** [comment]
Could you please try the "Mi Remote" and select "United" and "Strong" there , if you find a working (even partially working) code remember which one it was, i could try to use that to read the codes from my own Poco X3

**Comment by zorgoz** [comment]
I don't have STRONG in Mi Remote. UNITED does not work (none of the buttons)
![IMG_20210215_172421](https://user-images.githubusercontent.com/1569170/107973154-270f2600-6fb5-11eb-9500-e51bbac995ab.jpg)

**Comment by zorgoz** [comment]
These are from the interior of the remote. Google gave nothing relevant yet.

![IMG_20210216_191111](https://user-images.githubusercontent.com/1569170/108104999-55ac0000-708c-11eb-9822-929635d6ced6.jpg)
![IMG_20210216_191230](https://user-images.githubusercontent.com/1569170/108105007-5775c380-708c-11eb-96b1-e22454ea1a79.jpg)

**Comment by irplus-remote** [comment]
I don't think this will help unfortunately as the chip can be programmed in different ways. 
Do you have a raspberry pi for capturing the code maybe?

**Comment by zorgoz** [comment]
I do have a Pi. How can I capture the codes with it.

**Comment by irplus-remote** [comment]
yea here is a guide - if you encounter any problem let me know with the error
https://irplus-remote.github.io/#learning

**Comment by zorgoz** [comment]
I already thought about making something similar. My problem is that I don't know even which frequency to target - which sensor to procure. It might well not be 38Khz.

**Comment by irplus-remote** [comment]
You don't need to figure it out, most IR sensors for CIR will just capture any within 30-40KHz range :) It's most likely 38KHz anyway

**Comment by irplus-remote** [comment]
You might salvage a receiver from any broken device with a remote or just buy some TSOP ones for a few cents in a electronic shop
